src/everest_models/jobs/fm_compute_economics/parser.py

```diff
@@ -33,14 +33,17 @@ def build_argument_parser():
     required_group.add_argument(
         *CONFIG_ARGUMENT.split("/"),
         required=True,
-        type=partial(parse_file, schema=EconomicIndicatorConfig),
+        type=partial(parse_file, schema=EconomicIndicatorConfig)
```

Comment:
Does it exceed line limit or is there another reason this is not on a single line like the others?

Reply:
We are using `ruff-format` to format the code and that has a character limit per line. Any line longer than that will be reformated automatically.